src/dioptra/restapi/v1/entrypoints/schema.py

```diff
@@ -124,6 +138,10 @@ class EntrypointSnapshotRefSchema(EntrypointSnapshotRefBaseSchema):  # type: ign
         attribute="name",
         metadata=dict(description="Name of the Entrypoint resource."),
     )
+    id = fields.Integer(
```

Comment:
my comment is incorrect. Snapshot drops the normal ID. `id` was added here because it was checked as part of a response validation in a test. However, this test should be looking at the `snapshotId`